src/diagnostics/edmfx_diagnostics.jl

```diff
@@ -693,6 +749,20 @@ function compute_rhoaen!(
         out .= TD.air_density.(thermo_params, cache.precomputed.ᶜts⁰)
     end
 end
+function compute_rhoaen!(
```

Comment:
I don't think we need the diagnostics for environmental thermodynamic variables for diagnostic edmf as they are the same as grid-mean, and I don't think it is useful to save two variables that are exactly the same. Do you need them for anything?